GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/LooseObjectStepTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using GVFS.FunctionalTests.Tools;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace GVFS.FunctionalTests.Tests.EnlistmentPerFixture
+{
+    // TODO(Mac): Before these tests can be enabled PostFetchJobShouldComplete needs
+    // to work on Mac (where post-fetch.lock is not removed from disk)
+    [TestFixture]
+    [Category(Categories.FullSuiteOnly)]
+    [Category(Categories.MacTODO.M4)]
+    public class LooseObjectStepTests : TestsWithEnlistmentPerFixture
+    {
+        private const string TempPackFolder = "tempPacks";
+        private FileSystemRunner fileSystem;
+
+        // Set forcePerRepoObjectCache to true to avoid any of the tests inadvertently corrupting
+        // the cache 
+        public LooseObjectStepTests()
+            : base(forcePerRepoObjectCache: true)
+        {
+            this.fileSystem = new SystemIORunner();
+        }
+
+        private string GitObjectRoot

```

Comment:
these `get`s could be done with the `=>` syntax for brevity.

Reply:
Can you give the full example.  Sorry a bit confused here.

Reply:
```
private string GitObjectRoot => this.Enlistment.GetObjectRoot(this.fileSystem);
```